TrustModel.md

```diff
@@ -0,0 +1,426 @@
+- Open Web Application Security Project -
+  [Docker Security Threat Modeling](https://github.com/OWASP/Docker-Security/blob/main/001%20-%20Threats.md) [^4]
+
+
+A clear definition of trust and threats for the confidential containers project is needed to 
+ensure the components and architecture deliver the security principles expected for cloud native 
+confidential computing. It provides the solid foundations and unifying security principles 
+against which we can assess architecture and implementation ideas and discussions.
+
+This document highlights key concepts and background knowledge before progressing to an 
+overarching trust model and highlighting areas in which threat vectors are considered.
+
+Further documents will highlight more specific threat vectors in detail, considering risk, impact, 
+mitigation etc as the project progresses. The Security Assurance section of CNCF Cloud 
+Native Security Whitepaper[^2] will guide this more detailed threat vector effort.
+
+---
+
+## Personas
+Otherwise referred to as actors or threat agents, these are individuals or groups capable of 
+carrying out a particular threat. 
+In identifying personas we consider :
+- The Cloud Native Security Whitepaper Runtime Environment [^2] Figure 5. This highlights three 
+  layers, Cloud/Environment, Workload Orchestration, Application.
+- The Kubernetes overview of cloud native security [^3] identifies the 4C's of Cloud Native 
+  Security as Cloud, Cluster, Container and Code. However data is core to confidential 
+  containers rather than code.
+- The Technical Analysis of Confidential Computing [^1] defines Confidential Computing is the  
+  protection of data in use by performing computation in a hardware-based Trusted Execution Environment.
+
+In considering personas we recognise that a trust boundary exists between each persona and we
+explore how the least privilege principle (as described in Cloud Native security Paper, Page 36 
+[^2] ) should apply to any actions which cross these boundaries.
+
+ Confidential containers can provide enhancements to ensure that the 
+expected  code/containers are the only code that can operate over the data. However any  
+vulnerabilities within this code are not mitigated by using confidential containers, the Cloud  
+Native Security Whitepaper details Lifecycle aspects that relate to the security of the code 
+being placed into containers such as Static/Dynamic Analysis, Security Tests, Code Review etc 
+which must still be followed. 
+
+![Personas Model](./images/PersonaModel.svg)
+
+Anyone of these personas could attempt to perform malicious actions.
+
+### Cloud Operator
+This persona has privileges within the Cloud/Environment Infrastructure.
+They are responsible for availability of infrastructure used by cloud native environment
+- Has access to the physical hardware.
+- Has access to the process involved in deploying VSI's.
```

Comment:
Is VSI a standard term or should this read something like VMs or VM Images?

Reply:
Altered this to be more generic "Has access to the processes involved in deployment of compute/storage/memory used by any  orchestration components and by the workload."

I have addressed it in a PR specific to Trust Model Personas -> https://github.com/confidential-containers/documentation/pull/24
We can continue Trust Model Persona discussion in this new specific PR